### PR TITLE
Update docs for scheduler ephemeral storage

### PR DESCRIPTION
**Pull request by elena-kolevska**
Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**
- [ ] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://docs.dapr.io/contributing/contributing-overview/#developer-certificate-of-origin-signing-your-work))
- [ ] [Read the contribution guide](https://docs.dapr.io/contributing/docs-contrib/contributing-docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

### CODE REVIEW

File: daprdocs/content/en/operations/hosting/kubernetes/kubernetes-persisting-scheduler.md [hunk 1]
```diff
@@ -172,8 +172,8 @@ helm upgrade --install dapr dapr/dapr \
 
 ## Ephemeral Storage
 
-Scheduler can be optionally made to use Ephemeral storage, which is in-memory storage which is **not** resilient to restarts, i.e. all Job data will be lost after a Scheduler restart.
-This is useful for deployments where storage is not available or required, or for testing purposes.
+When running in non-HA mode, Scheduler can be optionally made to use Ephemeral storage, which is in-memory storage that is **not** resilient to restarts, i.e. all Job data will be lost after a Scheduler restart.
```
Comment (hhunter-ms) [inline daprdocs/content/en/operations/hosting/kubernetes/kubernetes-persisting-scheduler.md]:
```suggestion
When running in non-HA mode, Scheduler can be optionally made to use Ephemeral storage, which is in-memory storage that is **not** resilient to restarts. For example, all Job data will be lost after a Scheduler restart.
```

**Review by hhunter-ms — CHANGES_REQUESTED** [review]
tiny review

**Review by hhunter-ms — APPROVED** [review]

File: daprdocs/content/en/operations/hosting/kubernetes/kubernetes-persisting-scheduler.md [hunk 2]
```diff
@@ -172,8 +172,8 @@ helm upgrade --install dapr dapr/dapr \
 
 ## Ephemeral Storage
 
-Scheduler can be optionally made to use Ephemeral storage, which is in-memory storage which is **not** resilient to restarts, i.e. all Job data will be lost after a Scheduler restart.
-This is useful for deployments where storage is not available or required, or for testing purposes.
+When running in non-HA mode, Scheduler can be optionally made to use Ephemeral storage, which is in-memory storage that is **not** resilient to restarts. For example, all Job data will be lost after a Scheduler restart.
```
Comment (msfussell) [inline daprdocs/content/en/operations/hosting/kubernetes/kubernetes-persisting-scheduler.md]:
```suggestion
When running in non-HA mode, the Scheduler can be optionally made to use ephemeral storage, which is in-memory storage that is **not** resilient to restarts. For example, all jobs data is lost after a Scheduler restart.
```

File: daprdocs/content/en/operations/hosting/kubernetes/kubernetes-persisting-scheduler.md [hunk 3]
```diff
@@ -172,8 +172,8 @@ helm upgrade --install dapr dapr/dapr \
...
+This is useful for non-production deployments where storage is not available or required, or for testing purposes.
```
Comment (msfussell) [inline daprdocs/content/en/operations/hosting/kubernetes/kubernetes-persisting-scheduler.md]:
```suggestion
This is useful for non-production deployments where storage is not available or required, or for testing purposes.
```
```suggestion
This is useful in non-production deployments or for testing where storage is not available or required.
```

**Review by msfussell — COMMENTED** [review]
Also see comment here https://github.com/dapr/docs/pull/4452 in that we need to describe how to set up HA mode. If this is Kubernetes specific, then can be added to this article. Please coordinate with @cicoyle 

Also @hhunter-ms - All the headings in this article are upper case and need to be lower case. Maybe include this in another update or Elena if you can do this.

**Review by msfussell — APPROVED** [review]
LGTM

**Pull request merged**